[CI] IndexingPressureIT testWriteCanBeRejectedAtPrimaryLevel failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #12322 / windows-2025_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/qj6o5kzvvfd4o)
- [elasticsearch-periodic-platform-support #12322 / almalinux-10_platform-support-unix](https://gradle-enterprise.elastic.co/s/t7doqa2p7j5kq)
- [elasticsearch-periodic-platform-support #12322 / amazonlinux-2023_platform-support-aws](https://gradle-enterprise.elastic.co/s/tjokvjz222mns)
- [elasticsearch-periodic #12351 / openjdk21_checkpart1_java-fips-140-3-matrix](https://gradle-enterprise.elastic.co/s/b2qfixj5you5i)
- [elasticsearch-intake #40907 / part1](https://gradle-enterprise.elastic.co/s/h6us7jhb4caiw)
- [elasticsearch-pull-request #152377 / part-1](https://gradle-enterprise.elastic.co/s/reqkvpbh7mqqq)
- [elasticsearch-pull-request #151278 / part-1](https://gradle-enterprise.elastic.co/s/mtqldps476thw)

**Reproduction Line:**
```
gradlew ":server:internalClusterTest" --tests "org.elasticsearch.index.IndexingPressureIT.testWriteCanBeRejectedAtPrimaryLevel" -Dtests.seed=FCCBF65A1D1D84C1 -Dtests.locale=ff-Adlm-GM -Dtests.timezone=Asia/Chungking -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.index.IndexingPressureIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testWriteCanBeRejectedAtPrimaryLevel'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: a value greater than <22484L>
     but: <0L> was less than <22484L>
```

**Issue Reasons:**
- [main] 1 consecutive failures in test testWriteCanBeRejectedAtPrimaryLevel
- [main] 1 consecutive failures in step windows-2025_checkpart1_platform-support-windows
- [main] 1 consecutive failures in pipeline elasticsearch-periodic-platform-support
- [main] 7 failures in test testWriteCanBeRejectedAtPrimaryLevel (0.8% fail rate in 883 executions)
- [main] 2 failures in step part-1 (0.4% fail rate in 536 executions)
- [main] 2 failures in pipeline elasticsearch-pull-request (0.4% fail rate in 535 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -512,15 +512,6 @@ tests:
 - class: org.elasticsearch.xpack.stateless.shutdown.SigtermAzureRestartIT
   method: testRestart
   issue: https://github.com/elastic/elasticsearch/issues/150691
-- class: org.elasticsearch.index.IndexingPressureIT
-  method: testWriteCanBeRejectedAtPrimaryLevel
-  issue: https://github.com/elastic/elasticsearch/issues/150693
-- class: org.elasticsearch.index.IndexingPressureIT
-  method: testWriteCanBeRejectedAtCoordinatingLevel
-  issue: https://github.com/elastic/elasticsearch/issues/150694
-- class: org.elasticsearch.index.IndexingPressureIT
-  method: testWriteIndexingPressureMetricsAreIncremented
-  issue: https://github.com/elastic/elasticsearch/issues/150695
 - class: org.elasticsearch.indices.memory.breaker.HierarchyCircuitBreakerTelemetryIT
   method: testCircuitBreakerTripCountMetric
   issue: https://github.com/elastic/elasticsearch/issues/150700

